monitoring/grafana/grr_grafanalib_dashboards/reusable_panels.py

```diff
@@ -37,6 +37,92 @@ def avg_cpu_usage_percentage(grr_component):
         legendFormat="Average Process CPU Usage in %",
       ),
     ],
+    yAxes=YAxes(
+      left=YAxis(max=105)
+    ),
   )
 
-GENERAL_PANELS = [number_of_active_processes_graph, avg_cpu_usage_percentage]
+def sum_process_memory_bytes(grr_component):
+  return Graph(
+    title="Sum of Process Memory Bytes (across all instances)",
+    targets=[
+      Target(
+        expr='sum(process_resident_memory_bytes{{job="grr_{}"}})'.format(grr_component),
+        legendFormat="Resident Memory",
+      ),
+    ])
+
+def db_operations_latency(grr_component):
+  return Graph(
+    title="Database Operations Latency",
+    targets=[
+      Target(
+        expr='rate(db_request_latency_sum{{job="grr_{0}"}}[10m]) / rate(db_request_latency_count{{job="grr_{0}"}}[10m])'.format(grr_component),
+        legendFormat="Latency - Call: {{call}}",
+      ),
+    ])
+
+def db_operations_errors(grr_component):
+  return Graph(
+    title="Database Operations Errors Rate",
+    targets=[
+      Target(
+        expr='rate(db_request_errors_total{{job="grr_{0}"}}[10m])'.format(grr_component),
+        legendFormat="Operations Error Rate - Call: {{call}}",
+      ),
+    ])
+
+def threadpool_outstanding_tasks_vs_threads_num(grr_component):
+  return Graph(
+    title="Outstanding Tasks vs. Number of Threads",
+    targets=[
+      Target(
+        expr='sum(threadpool_outstanding_tasks{{job="grr_{}"}})'.format(grr_component),
+        legendFormat="Outstanding Tasks",
+      ),
+      Target(
+        expr='sum(threadpool_threads{{job="grr_{}"}})'.format(grr_component),
+        legendFormat="Threads",
+      ),
+    ])
+
+def threadpool_cpu_usa(grr_component):
+  return Graph(
+    title="Threadpool Average CPU Usage",
+    targets=[
+      Target(
+        expr='avg(rate(threadpool_cpu_use{{job="grr_{}"}}[30s])) * 100'.format(grr_component),
+        legendFormat="Average Process CPU Usage in % (over all jobs & pools)",
+      ),
+    ],
+    yAxes=YAxes(
+      left=YAxis(max=105)
+    ),
+  )
+
+def client_crashes(grr_component):
```

Comment:
Why is client_crashes reusable?

Reply:
The reason is that this metric is exposed by all GRR server components (Frontends, Workers and AdminUI). Since at the moment I can't manage to trigger the metric, I am not sure which one of them would be triggered (if any).

Reply:
Got it. Let's expose it on the frontend only - this seems like the most logical place to show crashes graph. I wouldn't show it on other dashboards.